### ISSUE TITLE
Videos with furigana/karaoke style subtitles

### ISSUE THREAD

**Issue by jam-madrigal** [arcusmaximus/YTSubConverter#91]
So, I followed the steps here and downloaded the subtitle files in srv3 format with yt-dlp, then converted them to .ass format with YTSubConverter. The English subs work great, whether i use the --visual option in command line or not, however, I can't seem to get subtitles with furigana to display properly. The furigana shows up like so

![image](https://github.com/arcusmaximus/YTSubConverter/assets/65047802/8495f56a-3e3f-4f75-bc58-f16f7ca5dd23)

When It should look like this
![image](https://github.com/arcusmaximus/YTSubConverter/assets/65047802/4ef912b2-48c0-4adc-bec1-bdc8e954c4ca)

Is there a way to correct this? 

I'm working with this video https://youtu.be/HOmS6Kn0bn8

**Comment by frezvent** [comment]
Ruby Text can only be displayed after conversion by the program, and will only be displayed as [.../..] in AegiSub.

**Comment by jam-madrigal** [comment]
> Ruby Text can only be displayed after conversion by the program, and will only be displayed as [.../..] in AegiSub.

Alright. Well, do you know of a way to take the existing srv3 file and being able to use it in a media player with the ruby text and karaoke timings similarly displayed as it is on youtube?

**Comment by frezvent** [comment]
Basically, the format for the ruby text feature is different from the default of Aegisub itself, and the one supported by this program. The program uses special override tags {yt...} including the ruby text tag {ytruby}.
If you want to display ruby text in Aegisub that has been converted from srv3, then you must change the entire subtitle line to follow Aegisub's default format. You can read this [tutorial](https://aegisub.org/docs/latest/furigana_karaoke/).

**Comment by arcusmaximus** [comment]
> Basically, the format for the ruby text feature is different from the default of Aegisub itself, and the one supported by this program. The program uses special override tags {yt...} including the ruby text tag {ytruby}

There's some confusion here. When creating Aegisub subtitles for conversion to SRV3, you indeed use `{\ytruby}` and the `[心/しん]` notation. When you perform a regular reverse conversion from SRV3 back to Aegisub, you again get that same notation. However, this ticket is about the visual reverse conversion, where the intention is very much to visually replicate the subtitles as they appear on YouTube, without requiring any manual actions by the user.

In short, this report is perfectly valid, and the only reason furigana isn't handled yet is that it's tricky to do. I'll see if I can work something out.

**Comment by vermian** [comment]
I'd also like this to be improved if possible. Same issue with this example video: [https://www.youtube.com/watch?v=EXM0EE_NpQc](url) 
Using the Japanese subtitle track. Thanks.
